helm-charts/charts/kube-starrocks/charts/operator/values.yaml

```diff
@@ -47,4 +47,7 @@ starrocksOperator:
     # - key: "key"
     #   operator: "Equal|Exists"
     #   value: "value"
-  #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+    #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+  # if specified, restricts the manager's cache to watch objects in the desired namespace.
+  # Defaults to all namespaces.
+  watchNamespace: ""
```

Comment:
what if want to watch multiple namespaces other than all the namespaces?

Reply:
k8s support to watch a single namespace or all namespaces. So we can not specify multiple clusters in `watchNamespace` field.

Reply:
that's sound like a risk option exposed to end user, if user have this field set, either set it wrong or the user has multiple clusters deployed in multiple namespace, the operator will not work as desired at all.

Please refine the message here, highlight the risk, if the user doesn't now what it means, don't set it at all.